css/main.css

```diff
@@ -3,18 +3,26 @@
 	--base-color: #E5D9B9;
 	--lighter-color: #FBEFD5;
 	--lightest-color: #FFF5B5;
-	--topbar-height: 2.5em;
 }
 
 html{
 	font-family: Calibri,serif;
-	scroll-padding-top: var(--topbar-height);/*below topbar*/
+	width: 100vw;
+	height: 100vh;
+	overflow: hidden;
 }
 
 body{
 	background-color: var(--base-color);
-	margin-top: 0;
+	margin: 0;
+	margin-left: 0.5em;
+	margin-right: 0.5em;
 	font-family: Calibri;
+	display: flex;
+	flex-direction: column;
+	overflow: hidden;
+	width: calc(100vw - 1em);
```

Comment:
ewwwwwwwwwww
this "100vw-1em" sucks because 1em is variable so your right margin will be inconsistent depending on zoom level/default text size/browser.

Reply:
https://youtu.be/PK2CwlPnECQ?t=8

I'll change this.

Reply:
I actually don't know why this or the iframe had widths less than 100%. I must've grandfathered in some css from before this overhaul without realizing what it did. I've removed both things and made the width 100% and it appears to have worked well.